### DISCUSSION TITLE
Avoid equalizing nvimtree and buffer sizes

### DISCUSSION THREAD

**Discussion by NotAShelf** [nvim-tree/nvim-tree.lua#2345] [Q&A]
From my understanding, the preserve_window_proportions setting is what I am looking for, but even when preserve_window_proportions is set to true the window is divided into two halves when a directory is opened with `vi .` and a file is selected in the filetree.

Am I missing a config option? If so, how can I avoid this and make the filetree open in its regular width

Current Behaviour
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/62766066/6ac6dced-a9ac-43d4-b53b-ee3c29efdce1)

Expected Behaviour
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/62766066/e55a6c1a-9861-4ef7-95b2-ef1dca5f1637)

**Comment by alex-courtis** [comment]
Setting `:h nvim-tree.view.preserve_window_proportions` to the default false should result in the behaviour you expect.

It could be that another plugin or setting is interfering. Please experiment with a [Clean Room Configuration](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#clean-room-replication) to see how all default settings behave.
